### PR TITLE
Fabien branch front

### PR DESCRIPTION
**Pull request by Respeccs**
Let me know if improvements are needed.

### CODE REVIEW

**Comment by TheRealPlagiariser** [comment]
Cuma mo rente lakaz mo chk li, add a description in regards to what you have done, I've linked the pull req with the current issue, Also you do not have to drag it from "in progress" to "done", this is automated once we do the merge.

**Comment by Respeccs** [comment]
- Added getClaimByUserID() in user,service.ts
- Subscribed to getClaimByUserID() in viewclaim,component.ts
- Added session storage in employee.component,ts and user.service.ts

File: src/app/services/users/users.service.ts [hunk 1]
```diff
@@ -12,6 +13,17 @@ export class UsersService {
   getUsers(): Observable<any> {
     return this.http.get(`${environment.apiUrl}/claims`);
   }
+
+  getClaimByUserID(UserData): Observable<any>{
+    let empid = sessionStorage.getItem('employeeId')
+    return this.http.post(`https://localhost:44382/GetClaimsById?employeeId=${empid}`, {UserData});
```
Comment (TheRealPlagiariser) [inline src/app/services/users/users.service.ts:19]: use environment declared in environment.ts, do not put api link here
(`${environment.apiUrl}/

Comment (TheRealPlagiariser) [inline src/app/services/users/users.service.ts:19]: use environment declared in environment.ts, do not put api link here
(`${environment.apiUrl}/

**Review by TheRealPlagiariser — COMMENTED** [review]
Once you have changed the link to environment, let me know so we can merge asap

**Pull request merged**